about.html

```diff
@@ -0,0 +1,130 @@
+                    of 2023. In the new trailer, Armin evokes the memories of three protagonist losistas when they were
+                    young, adding a layer of nostalgia to the plot. The phrase "Goodbye Ellen" spoken by Mikasa
+                    in the image key has generated speculation about the meaning suado. Fans expect twists
+                    surprises, shocking revelations and epic showdowns in this thrilling conclusion. He
+                    production team has worked hard to create a visually stunning experience and
+                    technically sound, while the writers have woven a cohesive and emotionally
+                    resonant. The second part " of The Final Season"ete promised to be a milestone in the history of
+                    anime and
+                    Fans are eager to describe how the final outcome will play out.
+                </p>
+                <p class="about-info-mail">If you want to contact the largest otaku community in Latin America</p>
+                <a href="#">
+                    <h4 class="about-mail">OtakuZone@gmail.com</h4>
+                </a>
+            </div>
+        </section>
+        <section class="about-section1">
+            <h2>why otakuzone?</h2>
+            <hr class="about-line">
+            <p>
+                This page was born as a tribute to my favorite anime SWORT ART ONLINE and the desire to create an online
+                space where
+                Fans can find informative and entertaining content about their favorite series and movies.
+                I realized there was a lack of reliable and comprehensive resources in the world of anime, and I wanted
+                to
+                fill that void. This page is for all anime lovers, from those who are new to
+                in this world even the most experienced fans.<br>Our goal is to provide information
+                information, reviews, analysis and news about the anime, as well as creating a community where fans
+                they can connect and share their love for this form of entertainment. Join us and immerse yourself in
+                the fascinating universe of anime.
+            </p>
+            <a href="index.html">
+                <img src="ressources/icons/logo.png" alt="">
+            </a>
+        </section>
+        <hr class="custom-style">
+        <section class="about-section1">
+            <h2>Crunchyroll Anime Awards 2023:</h2>
+            <hr class="about-line">
+            <p>The Crunchyroll Awards for Best Anime of the Year</p>
+            <section class="cont">
+                <div class="cont1">
+                    <h3>Best Anime</h3>
+                    <p>Cyberpunk: Edgerunners</p>
+                </div>
+                <div class="cont2">
+                    <h3>Best Action Anime</h3>
+                    <p>Kimetsu no Yaiba Pleasure Quarter Arc (Night's Watch, Demon Slayer)</p>
```

Comment:
- [optional] the footer of the mobile about page is missing the partners section and following the original design it has a white background. Kindly fix that to align with the original design.
yours
![Screenshot (1229)](https://github.com/AndresEspi/animePage/assets/65298048/9a672fc5-026a-4133-974f-7d0e1553374c)
Behance
![Screenshot (1168)](https://github.com/AndresEspi/animePage/assets/65298048/bc1112fd-3954-4a02-b5a9-76957899fe3a)